<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 43, 85);
color: rgb(103, 222, 144);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>641</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QMdiArea" name="mdiArea"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <widget class="QMenu" name="menuMantenimiento">
    <property name="font">
     <font>
      <family>PrimaSans BT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Mantenimiento</string>
    </property>
    <addaction name="actionClientes"/>
    <addaction name="actionProductos"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuGesti_n">
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Gesti√≥n</string>
    </property>
    <addaction name="actionFacturas"/>
   </widget>
   <widget class="QMenu" name="menuInformes">
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Informes</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="actionLista"/>
   </widget>
   <addaction name="menuMantenimiento"/>
   <addaction name="menuGesti_n"/>
   <addaction name="menuInformes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
  </widget>
  <action name="actionClientes">
   <property name="icon">
    <iconset>
     <normaloff>icons/cliente.png</normaloff>icons/cliente.png</iconset>
   </property>
   <property name="text">
    <string>Clientes</string>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="actionProductos">
   <property name="icon">
    <iconset>
     <normaloff>icons/carrito-de-compras.png</normaloff>icons/carrito-de-compras.png</iconset>
   </property>
   <property name="text">
    <string>Productos</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset>
     <normaloff>icons/salir.png</normaloff>icons/salir.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionFacturas">
   <property name="icon">
    <iconset>
     <normaloff>icons/factura.png</normaloff>icons/factura.png</iconset>
   </property>
   <property name="text">
    <string>Facturas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="actionLista">
   <property name="icon">
    <iconset>
     <normaloff>icons/imprimir.png</normaloff>icons/imprimir.png</iconset>
   </property>
   <property name="text">
    <string>Lista</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+L</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/app2.qrc"/>
 </resources>
 <connections/>
</ui>
